site/content/integrate/slash-commands/custom/_index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Custom commands
+heading: Custom slash commands
+weight: 20
+---
+Suppose you want to write an external application that is able to check the weather for certain cities. By creating a custom slash command and setting up the application to handle the HTTP `POST` or `GET` from the command, you can let your users check the weather in their city using your command, say `/weather toronto week`.
```

Comment:
"the weather in their city _for the week_ using your command"

Just to really make the slash command pop with clarity.